forge/lib/templates.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
         'modules_denyList',
         'httpNodeAuth_type',
         'httpNodeAuth_user',
-        'httpNodeAuth_pass'
+        'httpNodeAuth_pass',
+        'deviceAutoProvision'
```

Comment:
Is this a hangover from previous iterations? I don't see any relation to Templates for this feature.

Reply:
Resolved in [591c018](https://github.com/flowforge/flowforge/pull/1659/commits/591c018563622472a1dadcf7ae17ee23ead73efa)